backend/server.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+from controllers import file_controller
+from flask_cors import CORS
+import json
+import config
+
+app = Flask(__name__)
+api = Api(app)
+CORS(app)
+
+
+@app.route("/")
+class File(Resource):
+    def post(self):
+        response, response_code, filename = file_controller.post(request)
+        return app.response_class(
```

Comment:
Maybe consider to use the frameworks response generation (dict to json implicit conversion & status code)